core/blockchain.go

```diff
@@ -194,7 +196,7 @@ func (c *CacheConfig) triedbConfig() *triedb.Config {
 		config.DBOverride = hashdb.Config{
 			CleanCacheSize:                  c.TrieCleanLimit * 1024 * 1024,
 			StatsPrefix:                     trieCleanCacheStatsNamespace,
-			ReferenceRootAtomicallyOnUpdate: true, // Automatically reference root nodes when an update is made
+			ReferenceRootAtomicallyOnUpdate: true,
```

Comment:
nit: why remove the comment?

Reply:
Because it's not part of the master branch https://github.com/ava-labs/coreth/pull/820#discussion_r1984982667

Reply:
It's a revert commit, this was added in the libevm branch but not present in master branch, and I don't think it's related to libevm really right?